src/utils/isPRLabelValid.js

```diff
@@ -0,0 +1,19 @@
+// Ignore list of labels for frogtoberfest, add at end of array for more list
+const FROGTOBERFEST_IGNORE_LABEL_LIST = ['invalid'];
+
+// Makes The Array case insensitive for checking
+const labelIgnoreRegex = new RegExp(FROGTOBERFEST_IGNORE_LABEL_LIST.join('|'), 'i');
+
+/**
+ * Checks whether Pull request is valid against Frogtoberfest label list.
+ *
+ * @param {*} labels
+ */
+export default function isPRLabelValid(labels) {
```

Comment:
Don't need to have one function in one file. You can create a util file like `utils/pullRequest.js` and put all the functions related to PRs in that. Also you can do named export instead of default export here.